index.bs

```diff
@@ -52,15 +52,23 @@ This method allows web authors to register a callback which will run the next ti
 : <dfn for="VideoFrameMetadata" dict-member>expectedPresentationTime</dfn>
 :: The time at which the user agent expects the frame to be visible.
 
+Note: <dfn>media pixels</dfn> refers to the raw media pixels, after internal
```

Comment:
This concept is referenced from normative text, so it shouldn't defined in a note.

@tabatkins is something that Bikeshed could warn about?

Reply:
Oh def.